src/pathfinder/npf/aystar.h

```diff
@@ -20,6 +20,8 @@
 #include "../../tile_type.h"
 #include "../../track_type.h"
 
+static const int AYSTAR_MAX_SEARCH_NODES = 12312; ///< Reference limit for #AyStar::max_search_nodes
```

Comment:
Why not 10,000 like the setting default?

Also, this is not truly the max, it's the default setting for the max. Maybe a better name would be "AYSTAR_DEF_MAX_SEARCH_NODES".

Reply:
oh, my bad... lol I missed that. Thx for pointing out.